test/integration/resultsPagePostcode.js

```diff
@@ -88,7 +88,7 @@ describe('The results page', () => {
     const results = $('.results__item');
     expect(results.length).to.equal(numberOfResults);
 
-    const mapLinks = $('.results__maplink a');
+    const mapLinks = $('.maplink');
     mapLinks.toArray().forEach((link) => {
```

Comment:
See above comment

Reply:
Changes made and another test that wasn't doing anything fixed.